src/main/play-doc/operation/Install.md

```diff
@@ -951,6 +690,8 @@ ConductR comes with a `visualizer` sample application. Head over to the next sec
 
 The following guide will outline the steps to deploy and run ConductR as a framework within DC/OS.
 
+For ConductR installation specific to DC/OS built using Mesosphere's CloudFormation, refer to [DC/OS CloudFormation Installation](#DC/OS-CloudFormation-Installation)
```

Comment:
This guide presumes RHEL/CentOS or Debian nodes. CoreOS and those using [Mesosphere's AWS CloudFormation templates](https://dcos.io/docs/1.7/administration/installing/cloud/aws/) should refer to the [DC/OS CloudFormation Installation](#DC/OS-CloudFormation-Installation) section for EC2 specific instructions.

I'll be curious to see what installations in this not CoreOS category look like.